spec/controllers/state_file/questions/nj_household_rent_own_controller_spec.rb

```diff
@@ -69,14 +69,6 @@
         end
       end
 
-      context "when intake is both" do
-        let(:intake) { create :state_file_nj_intake, household_rent_own: "both" }
-
-        it "next path is unsupported page" do
-          expect(subject.next_path).to eq(StateFile::Questions::NjUnsupportedPropertyTaxController.to_path_helper)
-        end
-      end
-
```

Comment:
[dust/pebble] weakly held suggestion for keeping these tests to describe the new behavior (ineligible page) and changing them when that updates